README.md

```diff
@@ -359,6 +360,44 @@ This heading link of this heading is changed to following id
 - The `' '` (space) is replaced with `-` (dash), since `' '` is included in the first set
 - The '™' is replaced with _nothing_, since '™' is included in the second set
 
+#### URI encoding
+
+Non-ascii characters within link ids would be [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding) default.
```

Comment:
Rephrase: "By default non-ASCII characters in link ids are [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding)."